tests/wps_restapi/test_processes.py

```diff
@@ -178,20 +184,177 @@ def test_get_processes(self):
         resp = self.app.get(path, headers=self.json_headers)
         assert resp.status_code == 200
         assert resp.content_type == CONTENT_TYPE_APP_JSON
-        assert "processes" in resp.json and isinstance(resp.json["processes"], list) and len(resp.json["processes"]) > 0
+        assert "processes" in resp.json
+        assert isinstance(resp.json["processes"], list) and len(resp.json["processes"]) > 0
         for process in resp.json["processes"]:
             assert "id" in process and isinstance(process["id"], str)
             assert "title" in process and isinstance(process["title"], str)
             assert "version" in process and isinstance(process["version"], str)
             assert "keywords" in process and isinstance(process["keywords"], list)
             assert "metadata" in process and isinstance(process["metadata"], list)
-            assert len(process["jobControlOptions"]
-                       ) == 1 and EXECUTE_CONTROL_OPTION_ASYNC in process["jobControlOptions"]
+            assert len(process["jobControlOptions"]) == 1
+            assert EXECUTE_CONTROL_OPTION_ASYNC in process["jobControlOptions"]
 
         processes_id = [p["id"] for p in resp.json["processes"]]
         assert self.process_public.identifier in processes_id
         assert self.process_private.identifier not in processes_id
 
+    def test_get_processes_with_paging(self):
+        test_prefix = "test-proc-temp"
+        for i in range(10):
+            p_id = "{}-{}".format(test_prefix, i)
+            proc = self.process_private = Process(id=p_id, package={}, visibility=VISIBILITY_PUBLIC)
+            self.process_store.save_process(proc)
+        _, total = self.process_store.list_processes(total=True, visibility=VISIBILITY_PUBLIC)
+        assert 10 < total < 15, "cannot run process paging test with current number of processes"
```

Comment:
During test creation, the `process_store` is cleaned up (separate mongo db). 
It only contains the processes created during this `untittest.TestCase`. 
The guards here are just in case more tests get added over time and need adjustment of this test. 
It is easy to fix, but hard to understand the cause of error without this check.